data-plane/profiler/run.sh

```diff
@@ -101,7 +101,7 @@ export WAIT_STARTUP_SECONDS="8"
 export SERVICE_NAME="kafka-broker-receiver"
 export SERVICE_NAMESPACE="knative-eventing"
 export INGRESS_PORT="8080"
-export METRICS_PORT="9090"
+export METRICS_PORT="9098"
 export INSTANCE_ID="receiver"
```

Comment:
Why the port # change?

Reply:
Because sacura started to expose metrics on 9090, so that broke this job :)

Reply:
:thinking:

Reply:
I started to expose metrics on 9090 for perf testing on sacura side, so the receiver and sacura are trying to use the same local port 

https://github.com/pierDipi/sacura/commit/8d35ca8ae2beb65c7e5764d5d25efa0a018f30b4